api/comments.rb

```diff
@@ -62,5 +62,4 @@
       pass
     end
   end
-  comment.to_hash.to_json
```

Comment:
I removed this line because it didn't make much sense (to me) to return the body of a deleted object.